packages/itwinui-react/src/core/Surface/Surface.tsx

```diff
@@ -29,7 +29,47 @@ const getSurfaceElevationValue = (elevation: SurfaceProps['elevation']) => {
   }
 };
 
+// ----------------------------------------------------------------------------
+// Surface.Header component
+
+export type SurfaceHeaderProps = React.ComponentPropsWithRef<'div'>;
+
+const SurfaceHeader = React.forwardRef<HTMLDivElement, SurfaceHeaderProps>(
+  (props, ref) => {
+    const { children, className, ...rest } = props;
+    return (
+      <div className={cx('iui-surface-header', className)} ref={ref} {...rest}>
+        {children}
+      </div>
+    );
+  },
+);
+
+// ----------------------------------------------------------------------------
+// Surface.Body component
+
+export type SurfaceBodyProps = React.ComponentPropsWithRef<'div'>;
+
+const SurfaceBody = React.forwardRef<HTMLDivElement, SurfaceBodyProps>(
```

Comment:
these should all be polymorphic

you can check how to do that in `Flex`

Reply:
These have been changed so they are polymorphic

Reply:
Did you check if they are really polymorphic?
Update unit tests to include this check.

Reply:
I have now checked that the component is polymorphic and have added a unit test for it